tests/test_derivatives.py

```diff
@@ -1,44 +1,65 @@
 import numpy as np
 import pytest
-from conftest import t, x, y, z, skipif_yask
+from conftest import skipif_yask
 from sympy import Derivative, simplify
 
 from devito import Grid, Function, TimeFunction
 
 
 @pytest.fixture
-def shape(xdim=20, ydim=30):
-    return (xdim, ydim)
+def shape(xdim=20, ydim=20, zdim=20):
```

Comment:
Reason for changing `ydim`? There are merits to having non square/cube domains in tests.

Reply:
OK